OrbitGl/TracepointTrack.cpp

```diff
@@ -69,14 +69,38 @@ void TracepointTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
 
   const Color kWhite(255, 255, 255, 255);
 
+  const Color kTracepoint(255, 255, 255, 190);
+
   const Color kGreenSelection(0, 255, 0, 255);
 
   if (!picking) {
     for (auto it = tracepoints.lower_bound(min_tick); it != tracepoints.upper_bound(max_tick);
          ++it) {
       uint64_t time = it->first;
       Vec2 pos(time_graph_->GetWorldFromTick(time), pos_[1]);
-      batcher->AddVerticalLine(pos, -track_height, z, kWhite);
+      batcher->AddVerticalLine(pos, -track_height / 4, z, kTracepoint);
```

Comment:
I guess it is easier and faster to just draw the entire line. The part in the middle will be overdrawn by the circle anyway.

Reply:
Not really, I have tried it and because of the alphas the line in the circle gets lighter and visible. @pierricgimmig  may know more about this because he suggested to split the line.

Reply:
Yeah, because the circle has transparency, drawing a single line looks weird...